src/include/ctype.i

```diff
@@ -0,0 +1,139 @@
+/*-
+ * Copyright (c) 2014-2016 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <ctype.h>
+
+/*
+ * __wt_isalnum --
+ *	Wrap the ctype function without sign extension.
+ */
+static inline int
+__wt_isalnum(int c)
```

Comment:
Can our wrappers take a `u_char` argument to avoid the explicit cast?  That seems more likely to generate warnings from invalid code...

I'm not so fussed about the `int` return here, but we could consider returning a `bool`.  In other words, this would become:

```c
static inline bool
__wt_isalnum(u_char c)
{
        return (isalnum(c) != 0);
}
```